languages/haskell/config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "language": "Haskell",
+  "version": 3,
+  "active": true,
+  "blurb": "Haskell is a functional programming language which is pure and statically-typed. It's known for lazy evaluation, where evaluation is deferred until necessary, and its purity, where monads are used for working with side-effects.",
+  "online_editor": {
+    "indent_style": "space",
+    "indent_size": 2
+  },
+  "ignore_pattern": "example",
+  "solution_pattern": "example.*[.]hs",
+  "exercises": {
+    "concept": [],
+    "practice": []
+  }
```

Comment:
It looks like the "foregone" exercises have also been removed. See https://github.com/exercism/haskell/blob/dc01f701596c6726b4c6f31acf9e97df840139ef/config.json#L7. Was that intentional?

Reply:
Sort of.

I deliberately didn't add the exercise-specific data because [migrating-your-config-json-files.md](https://github.com/exercism/v3/blob/master/docs/maintainers/migrating-your-config-json-files.md) says:

> The existing exercises are temporarily removed from the `config.json` file. They will return as practice exercises once the concept exercises have been added.

I thought it was slightly strange to mention which practice exercises have been foregone when not mentioning which practice exercises that are implemented. So I thought I would add foregone exercises after I've added practice exercises after I've added 20+ concept exercises. At least that's how I understood the recommended workflow.

Reply:
You are entirely correct of course. I'll update the migrate guide to also mention removing the `foregone` key.